{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-16T07:34:00.000+0000",
      "latitude": 50.899510815312276,
      "estimatedRestoration": "2022-04-16T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR8511'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR8511",
      "reference": "JR8511",
      "engineerOnSiteTime": "2022-04-16T08:31:00.000+0000",
      "name": "SO14 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO14 3EW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.3945837011855922,
          50.899510815312276
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-16T10:15:00.000+0000",
      "UUID": "5cf27e7b-388c-4806-bbcf-bbdada4fe4e1",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3945837011855922,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-16T11:48:00.000+0000",
      "latitude": 50.858817826985586,
      "estimatedRestoration": "2022-04-16T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR8592'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR8592",
      "reference": "JR8592",
      "name": "PO18 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO18 0FF",
        "PO18 0HR",
        "PO18 0LA",
        "PO18 0LB",
        "PO18 0NF",
        "PO18 0NL",
        "PO18 0NR",
        "PO18 0NS",
        "PO18 0NT",
        "PO18 0NU",
        "PO18 0NW",
        "PO18 0NX",
        "PO18 0PA",
        "PO18 0PF",
        "PO18 0PG",
        "PO18 0PH",
        "PO18 0PJ",
        "PO18 0PL",
        "PO18 0PN",
        "PO18 0PP",
        "PO18 0PQ",
        "PO18 0PR",
        "PO18 0PW",
        "PO18 0PX",
        "PO18 0PY",
        "PO18 0QA",
        "PO18 0QB",
        "PO18 0QR",
        "PO18 0QU",
        "PO18 0RN",
        "PO18 0RY",
        "PO18 0RZ",
        "PO18 0SF",
        "PO18 0TU",
        "PO19 7QL",
        "PO19 7RT",
        "PO20 2AG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.7445789092022332,
          50.858817826985586
        ],
        "type": "Point"
      },
      "UUID": "e6686cb2-1751-4bac-bbd8-ab3c91191f28",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "4d056d17-c156-4124-b380-99750e2baa08",
        "78fe4730-856f-4c4f-b5c9-6357463c8fed"
      ],
      "longitude": -0.7445789092022332,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-16T11:32:00.000+0000",
      "latitude": 51.60175018634854,
      "estimatedRestoration": "2022-04-16T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR8588'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR8588",
      "reference": "JR8588",
      "engineerOnSiteTime": "2022-04-16T12:30:00.000+0000",
      "name": "HP9 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP10 0NN",
        "HP9 1DG",
        "HP9 1DJ",
        "HP9 1DQ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.6752952263016637,
          51.60175018634854
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-16T12:30:00.000+0000",
      "UUID": "41146ddc-2f86-4f39-a1e9-8ee505d851b4",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6752952263016637,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-16T13:49:00.000+0000",
      "latitude": 50.63193813952645,
      "estimatedRestoration": "2022-04-16T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR8627'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR8627",
      "reference": "JR8627",
      "name": "PO37 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO37 6AT",
        "PO37 6AW",
        "PO37 6AY",
        "PO37 6BA",
        "PO37 6DB",
        "PO37 6DD",
        "PO37 6DE",
        "PO37 6DF",
        "PO37 6DG",
        "PO37 6DH",
        "PO37 6DJ",
        "PO37 6DL",
        "PO37 6DN",
        "PO37 6DP",
        "PO37 6DQ",
        "PO37 6DR",
        "PO37 6DW",
        "PO37 6EA",
        "PO37 6ED",
        "PO37 6EJ",
        "PO37 6FA",
        "PO37 7AF",
        "PO37 7AG",
        "PO37 7AH",
        "PO37 7AN",
        "PO37 7AQ",
        "PO37 7AT",
        "PO37 7AU",
        "PO37 7AX",
        "PO37 7BF",
        "PO37 7BQ",
        "PO37 7BT"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.175541845568924,
          50.63193813952645
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-16T14:30:00.000+0000",
      "UUID": "05cf9368-0579-4445-9e0c-32affd3e7bf3",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.175541845568924,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-16T14:06:00.000+0000",
      "latitude": 51.04397209764026,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR8637'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR8637",
      "reference": "JR8637",
      "name": "SO22 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO21 2AY",
        "SO21 2JS",
        "SO22 4BF",
        "SO22 4HB",
        "SO22 4HD",
        "SO22 4HE",
        "SO22 4HF",
        "SO22 4HG",
        "SO22 4HH",
        "SO22 4HJ",
        "SO22 4HL",
        "SO22 4HN",
        "SO22 4HP",
        "SO22 4HQ",
        "SO22 4HR",
        "SO22 4HS",
        "SO22 4HT",
        "SO22 4HU",
        "SO22 4HW",
        "SO22 4JL",
        "SO22 4JN",
        "SO22 4JP",
        "SO22 4JR",
        "SO22 4JS",
        "SO22 4JT",
        "SO22 4JU",
        "SO22 4JW",
        "SO22 4JX",
        "SO22 4JY",
        "SO22 4JZ",
        "SO22 4LA",
        "SO22 4LB",
        "SO22 4LD",
        "SO22 4LE",
        "SO22 4LF",
        "SO22 4QB",
        "SO22 4QR",
        "SO22 4QS",
        "SO22 5QN",
        "SO22 5QP",
        "SO22 5QR",
        "SO22 5QW",
        "SO22 5QX",
        "SO22 5SW",
        "SO23 9RZ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.3507660194893334,
          51.04397209764026
        ],
        "type": "Point"
      },
      "UUID": "27195408-cde3-4337-a4b6-6505edabf462",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "67156fe4-a843-454d-ab88-236ca9c8c680"
      ],
      "longitude": -1.3507660194893334,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-14T13:00:00.000+0000",
      "latitude": 50.756511631768824,
      "estimatedRestoration": "2022-04-16T14:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR7753'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR7753",
      "reference": "JR7753",
      "name": "BH23 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH23 5QB",
        "BH23 7EA",
        "BH23 7EH",
        "BH23 7EJ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.6941990265267624,
          50.756511631768824
        ],
        "type": "Point"
      },
      "UUID": "81b68ecc-ab03-469d-ad56-b421f3043ddd",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.6941990265267624,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-16T12:42:00.000+0000",
      "latitude": 57.42558723097245,
      "estimatedRestoration": "2022-04-16T15:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR8608'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR8608",
      "reference": "JR8608",
      "name": "HS7 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "HS7 5PJ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -7.358541383948576,
          57.42558723097245
        ],
        "type": "Point"
      },
      "UUID": "94947544-afda-4f00-a6e1-15c651d856d4",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -7.358541383948576,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-16T12:21:00.000+0000",
      "latitude": 57.53773259250123,
      "estimatedRestoration": "2022-04-16T15:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR8601'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR8601",
      "reference": "JR8601",
      "name": "AB55 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "AB55 5AB",
        "AB55 5AG",
        "AB55 5AH",
        "AB55 5AJ",
        "AB55 5AU",
        "AB55 5BQ",
        "AB55 5GG",
        "AB55 5HG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.9506182147009676,
          57.53773259250123
        ],
        "type": "Point"
      },
      "UUID": "25677041-ce4c-46fa-8681-f96ed4f79120",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "f87990a4-2c75-4126-bfa6-d0d0d355dae8"
      ],
      "longitude": -2.9506182147009676,
      "resolved": false
    }
  ]
}
